src/Formatter/Formatter.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\FormatterBundle\Formatter;
+
+interface Formatter
+{
+    public function transform(string $text);
```

Comment:
Maybe we could tell that with the next major, `void` will be declared as return type.